src/main/java/org/dita/dost/invoker/Main.java

```diff
@@ -135,6 +135,22 @@ String getValue(final String value) {
         }
     }
 
+    private static class AbsoluteFileListArgument extends Argument {
+        AbsoluteFileListArgument(final String property) {
+            super(property);
+        }
+
+        @Override
+        String getValue(final String value) {
+            String allPaths = "";
```

Comment:
This should really use a stream that's mapped and then collected with a joiner, but I'm ok with rewriting it later on.